frontend/app/forms/proof-of-completed-service/proof-of-completed-service.pdf.component.js

```diff
@@ -1,5 +1,149 @@
+          top={countyDistrictTop}
+          debugKey="county"
+        >
+          {data.case.courtAddress &&
+            getCounty(data.case.courtAddress, data.case.courtType)}
+        </PositionedString>
+
+        <PositionedString
+          left="25.9%"
+          top={countyDistrictTop}
+          debugKey="judicialDistrict"
+        >
+          {data.case.courtAddress &&
+            getJudicialDistrict(data.case.courtAddress, data.case.courtType)}
+        </PositionedString>
+
+        <PositionedString
+          dataKey="case.courtAddress"
+          left="27.8%"
+          top="39.8%"
+        />
+
+        <PositionedCheckmark
+          datakey="case.summons"
+          left={documentsServedCheckmarks}
+          top="68.4%"
+          shouldShow={data.case.summons}
+        />
+        <PositionedCheckmark
+          datakey="case.complaintOrPetition"
+          left={documentsServedCheckmarks}
+          top="70.4%"
+          shouldShow={data.case.complaintOrPetition}
+        />
+        <PositionedCheckmark
+          datakey="case.amendedComplaintOrPetition"
+          left={documentsServedCheckmarks}
+          top="72.4%"
+          shouldShow={data.case.amendedComplaintOrPetition}
+        />
+        <PositionedCheckmark
+          datakey="case.noticeOfDivorceEducationRequirements"
+          left={documentsServedCheckmarks}
+          top="74.4%"
+          shouldShow={data.case.noticeOfDivorceEducationRequirements}
+        />
+        <PositionedCheckmark
+          datakey="case.noticeOfURCP261"
+          left={documentsServedCheckmarks}
+          top="76.4%"
+          shouldShow={data.case.noticeOfURCP261}
+        />
+        <PositionedCheckmark
+          datakey="case.noticeOfURCP263"
+          left={documentsServedCheckmarks}
+          top="80.4%"
+          shouldShow={data.case.noticeOfURCP263}
+        />
+        <PositionedCheckmark
+          datakey="case.parentingPlan"
+          left={documentsServedCheckmarks}
+          top="82.4%"
+          shouldShow={data.case.parentingPlan}
+        />
+        <PositionedCheckmark
+          datakey="case.other"
+          left={documentsServedCheckmarks}
+          top="84.4%"
+          shouldShow={data.case.other}
+        />
+        <PositionedString
+          dataKey="case.otherDocumentsServed"
+          left="28.1%"
+          top="84.1%"
+        />
+      </RenderPage>
+      <RenderPage url="/static/forms/proof-of-completed-service/proof_of_compeleted_service-2.png" />
+      <RenderPage url="/static/forms/proof-of-completed-service/proof_of_compeleted_service-3.png" />
+      <RenderPage url="/static/forms/proof-of-completed-service/proof_of_compeleted_service-4.png" />
+      <RenderPage url="/static/forms/proof-of-completed-service/proof_of_compeleted_service-5.png" />
+    </>
+  );
 }
+const personalInfo = `11.5%`;
+const courtTypeTop = `33.5%`;
+const countyDistrictTop = `36.65%`;
+const documentsServedCheckmarks = `19.64%`;
```

Comment:
typically my personal preference would be to declare these above the function, so like on line 11. That said, are we doing them below elsewhere in the app? if so It's fine.